tests/Constraints/FactoryTest.php

```diff
@@ -76,42 +81,42 @@ public function constraintNameProvider()
      * @dataProvider invalidConstraintNameProvider
      *
      * @param string $constraintName
-     * @return void
      */
     public function testExceptionWhenCreateInstanceForInvalidConstraintName($constraintName)
     {
         $this->setExpectedException('JsonSchema\Exception\InvalidArgumentException');
         $this->factory->createInstanceFor($constraintName);
     }
 
-    public function invalidConstraintNameProvider() {
-      return array(
+    public function invalidConstraintNameProvider()
+    {
+        return array(
         array('invalidConstraintName'),
       );
     }
 
     /**
-     * @expectedException InvalidArgumentException
+     * @expectedException \InvalidArgumentException
```

Comment:
It's not the current namespace, but it's probably what was intended, as convention everywhere else in the project is to use the namespaced exceptions. This one is less "the auto-fixer screwed up", and more "the original source screwed up, and the auto-fixer perpetuated that, so we should fix now it rather than making the error worse".